libs/services/NLM.py

```diff
@@ -0,0 +1,41 @@
+from io import StringIO
+import pandas as pd
+
+from libs.services.Converter import Converter
+
+
+class NLM(Converter):
+    def __init__(self):
+        super().__init__()
+        # service URLs
+        self.services = {'NLM': 'https://chem.nlm.nih.gov/api/data/'}
+
+    def inchikey_to_name(self, inchikey):
+        """
+        Convert InChiKey to Chemical name using NLM service
+        More info: https://chem.nlm.nih.gov/chemidplus/inchikey
```

Comment:
The standard sized file is a difficult question: Might range from couple thousand entries to multiple hundred thousand.

AFAIK, none of the currently queried APIs has any limitations on that.

Reply:
I'm not sure about this, but that's the reason I introduced cache. Maybe @hechth knows more about this?

In this version, the caching is not limited. Should be improved in #8 using `lru_cache` approach.

Reply:
How many http requests are we going to perform on a file that has 100k entries?

Reply:
Some of the services can respond with all available data instead of a single target attribute value (does not hold in general, though). This, in combination with caching, might help the issue, although it won't solve it completely.

Reply:
Let's continue in https://github.com/RECETOX/pyMSPannotator/issues/11